### PR TITLE
ADD bear65 V2 keyboard

### PR DESCRIPTION
**Pull request by 704340378**
## Types of Changes

- [x] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bear65/config.h [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+#define LOCKING_RESYNC_ENABLE
+
+
+
+
+#define RGB_DI_PIN E2
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_LIMIT_VAL 200
+#define RGBLIGHT_DEFAULT_VAL 50
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/bear65/config.h:57]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

File: keyboards/bear65/Makefile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ifndef MAKEFILE_INCLUDED
```
Comment (drashna) [inline keyboards/bear65/Makefile:1]: This file is long deprecated and needs to be removed.

File: keyboards/bear65/config.h [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+/* number of backlight levels */
+#define LED_CAPS_LOCK_PIN C7
+#define LED_PIN_ON_STATE 0
+#define BACKLIGHT_PIN B6
+#ifdef BACKLIGHT_PIN
+#define BACKLIGHT_LEVELS 3
+#endif
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
```
Comment (drashna) [inline keyboards/bear65/config.h:42]:
```suggestion
#define DEBOUNCE 5
```

File: keyboards/bear65/readme.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# bear65 v2
+
+![bear65 V2](https://i.imgur.com/ZsNn4gS.jpg)
```
Comment (drashna) [inline keyboards/bear65/readme.md:3]:
```suggestion
![bear65 V2](https://i.imgur.com/ZsNn4gSh.jpg)
```

File: keyboards/bear65/readme.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+A 65% keyboard kit sold through Drop.
+
+Keyboard Maintainer: QMK Community  
+Hardware Supported: Bear65 v2 PCB
```
Comment (drashna) [inline keyboards/bear65/readme.md:8]:
```suggestion
* Keyboard Maintainer: QMK Community
* Hardware Supported: Bear65 v2 PCB
```

Also: 
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/bear65/rules.mk [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+#
+BOOTMAGIC_ENABLE = yes	# Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no	# Mouse keys
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+CONSOLE_ENABLE = no	# Console for debug
+COMMAND_ENABLE = no    # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no  # Enable keyboard backlight functionality
+AUDIO_ENABLE = no
+RGBLIGHT_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/bear65/rules.mk:16]:
```suggestion

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = no            # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/bear65/rules.mk [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# MCU name
+MCU = atmega32u4
+
+BOOTLOADER = atmel-dfu
```
Comment (fauxpark) [inline keyboards/bear65/rules.mk:4]:
```suggestion
# Bootloader selection
BOOTLOADER = atmel-dfu
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Pull request closed without merging**